libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookAdapter.cs

```diff
@@ -63,25 +69,25 @@ public override async Task<ResourceResponse[]> SendActivitiesAsync(ITurnContext
             {
                 if (activity.Type != ActivityTypes.Message)
                 {
-                    throw new Exception("Only Activities of type Message are supported for sending.");
+                    _logger.LogTrace($"Unsupported Activity Type: '{activity.Type}'. Only Activities of type ‘Message’ are supported.");
```

Comment:
use straight or double quotes: 'Message' or "Message", the smart quotes ‘’ may not render well in some devices